src/cljs/athens/style.cljs

```diff
@@ -175,7 +176,7 @@
 
 (defn init
   []
-  (stylefy/init)
+  (stylefy/init {:dom (stylefy-reagent/init)})
```

Comment:
What's `stylefy-reagent` do?

Reply:
Stylefy 3.0 is broken up into separate packages, one for reagent, one for fulcro. I updated stylefy to see if I could address a styling syntax issue, but it turns out to have not been necessary for that. Stylefy update could be removed from this PR if desired.

Reply:
Seems like we shouldn't import `stylefy/reagent` until we know we need it.